### DISCUSSION TITLE
org mode suport

### DISCUSSION THREAD

**Discussion by kascesar** [quarto-dev/quarto-cli#10282] [Feature Requests]
### Description

Working with markdown files with Quarto are flawlessly. Recent years i use **Emacs**. I was used org-mode from emacs, thats work like markdown. So for that, y get the idea (and i want) to use org files to like qmd files without to export mi org files to qmd files. there is a possibility ? what you think about ?

**Comment by mcanouil** [comment]
Input for Quarto is markdown and Jupyter Notebook (it uses markdown).
Other inputs have to be converted and that is unlikely to change, especially if the format is not supported by Pandoc (https://pandoc.org).

It seems Pandoc supports org-mode so you could leverage Pandoc to convert all your files to Markdown.
